pkg/gateway/stats.go

```diff
@@ -7,7 +7,8 @@ import (
 
 var requestHistograms = promauto.NewHistogramVec(
 	prometheus.HistogramOpts{
-		Name: "gateway_request_duration_seconds",
-		Help: "request durations for lakeFS storage gateway",
+		Name:    "gateway_request_duration_seconds",
+		Help:    "request durations for lakeFS storage gateway",
+		Buckets: []float64{0.01, 0.05, 0.1, 0.2, 0.25, 0.5, 1, 2.5, 5, 10, 30, 60},
```

Comment:
Not a fan of the intervals, especially the middle octave 0.1, 0.2, 0.25, 0.5, 1.0.  The bucket-to-bucket size ratios are really weird!

| limit | 0.01 | 0.05 | 0.1 | 0.2 | 0.25 | 0.5 | 1 | 2.5 | 5 | 10 | 30 | 60 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| next limit / this limit | - | 5.0 | 2.0 | 2.0 | 1.25 | 2.0 | 2.0 | 2.5 | 2.0 | 2.0 | 3.0 | 2.0 |

The jumps 0.1 -> 0.2 -> 0.25 -> 0.5 are pretty poorly spaced: 2x, 1.25x, 2x.  I would get rid of 0.2.

I don't care that the spacing on the buckets >10s is weird, it's not as different and the numbers there will barely matter (until they do).

```suggestion
		Buckets: []float64{0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 30, 60},
```

Reply:
@arielshaqed, I changed both here and in the API, PTAL